lib/pbio/drv/legodev/legodev_pup.c

```diff
@@ -314,62 +330,38 @@ static void debug_state_change(ext_dev_t *dev) {
 #define debug_state_change(...)
 #endif
 
-PROCESS_THREAD(pbio_legodev_pup_process, ev, data) {
-    static struct etimer timer;
+static PT_THREAD(pbdrv_legodev_pup_thread(ext_dev_t * dev)) {
 
-    static ext_dev_t *dev;
+    PT_BEGIN(&dev->pt);
```

Comment:
It might save a few byte in the firmware to make a few local variables like `struct *pt = &dev->pt` to avoid multiple dereferences of the same pointer.

Reply:
it gets 12 byte bigger